src/js/libcs/Parser.js

```diff
@@ -171,21 +171,21 @@ function anyOfGroup(lst) {
     return "(" + lst.map(rescape).join("|") + ")";
 }
 
-var supDigit = "[⁰¹²³⁴⁵⁶⁷⁸⁹]";
-var subDigit = "[₀₁₂₃₄₅₆₇₈₉]";
-var supNum = expandSpaces("(?:[⁺⁻] )?" + supDigit + "(?: " + supDigit + ")*");
-var subNum = expandSpaces("(?:[₊₋] )?" + subDigit + "(?: " + subDigit + ")*");
+const supDigit = "[⁰¹²³⁴⁵⁶⁷⁸⁹]";
+const subDigit = "[₀₁₂₃₄₅₆₇₈₉]";
+const supNum = expandSpaces("(?:[⁺⁻] )?" + supDigit + "(?: " + supDigit + ")*");
+const subNum = expandSpaces("(?:[₊₋] )?" + subDigit + "(?: " + subDigit + ")*");
 
 // Letters (Unicode 8.0.0 category L), or rather their UTF-16 encoding
 // (using surrogate pairs as needed). Generated by tools/unicodeCat.js.
 // See Parser_test.js for an uncompressed version of this string.
-var unicodeLetters = (function (dict, str, hiRanges) {
-    var i, fst;
-    var j = 0;
-    var res = "(?:[";
-    var n = str.length;
+const unicodeLetters = (function (dict, str, hiRanges) {
+    let i, fst;
+    let j = 0;
+    let res = "(?:[";
+    const n = str.length;
     for (i = 0; i < n; ++i) {
```

Comment:
Change `for (i = 0;` to `for (let i = 0;` and remove `i` from `let` statement above.